unittests/AST/ASTImporterTest.cpp

```diff
@@ -1996,6 +1996,193 @@ TEST_P(ASTImporterOptionSpecificTestBase, ReimportWithUsedFlag) {
   EXPECT_TRUE(Imported2->isUsed(false));
 }
 
+// FIXME Can we unify these tests with ImportClassTemplates?
```

Comment:
I could unify these tests similarly to the Visibility tests in #502 , so they would be both type and value parameterized tests.  In one hand we will not have that much repetition of the test code, but on the other hand understanding it will be more difficult. @balazske what do you think?

Reply:
I think in case of `ASTImporter` there are many places for test generalization because many similar test cases, so it is better to have general tests that work with every possible parameter combination. Otherwise we can have too many test cases.